addon/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "tracked-toolbox",
+  "version": "1.2.3",
+  "description": "Helpful autotracking utilities",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:tracked-tools/tracked-toolbox.git"
+  },
+  "license": "MIT",
+  "author": "",
+  "exports": {
+    ".": "./dist/index.js",
+    "./addon-main.js": "./addon-main.js"
+  },
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "eslint . --cache",
+    "lint:fix": "eslint . --fix",
+    "prepublishOnly": "cp ../README.md ../LICENSE.md . && rollup --config",
+    "start": "rollup --config --watch"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.6.0",
+    "ember-cache-primitive-polyfill": "^1.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.9",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-decorators": "^7.17.9",
+    "@embroider/addon-dev": "^1.6.0",
+    "@rollup/plugin-babel": "^5.3.1",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^10.6.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "^2.6.2",
+    "rollup": "^2.70.2"
+  },
+  "peerDependencies": {
+    "ember-source": "*"
+  },
+  "peerDependenciesMeta": {
+    "ember-source": {
+      "optional": true
+    }
+  },
+  "engines": {
+    "node": "12.* || 14.* || >= 16"
```

Comment:
we can drop Node.js 12 support in separate PR as it's End of Life tomorrow (as of date of this writing)